.github/workflows/ci.yml

```diff
@@ -10,7 +10,6 @@ jobs:
     name: Pre-test checks
     runs-on: ubuntu-latest
     strategy:
-      max-parallel: 4
```

Comment:
What's the motivation for removing `max-parallel` from all the matrix jobs?

Reply:
We couldn't remember why it was added in the first place, and it was slowing down the run. We might as well take as many parallel jobs as GitHub is willing to give us.

Reply:
Ah, it *removes* the limit - that makes sense then. For some reason I assumed it would default to no parallelization at all.